metaswitch/common/pdlogs.py

```diff
@@ -60,8 +60,9 @@ def log(self, **kwargs):
     desc="The connection to Cassandra has been lost.",
     cause="The connection to Cassandra has been lost.",
     effect="The node can no long offer services that rely on Cassandra.",
```

Comment:
Hmm, good point, this log is in the context of a service (although I don't think we ever have more than one service running that can generate this).